services/brig/src/Brig/Team/API.hs

```diff
@@ -171,6 +174,10 @@ createInvitation (_ ::: uid ::: _ ::: tid ::: req) = do
     blacklisted <- lift $ Blacklist.exists uk
     when blacklisted $
         throwStd blacklistedEmail
+    maxSize <- setMaxTeamSize <$> view settings
+    pending <- lift $ DB.countInvitations tid
+    when (fromIntegral pending >= maxSize) $
```

Comment:
This could be improved to `when (fromIntegral pending + (# team members) >= maxSize)`, but would require an extra request to galley.